migrate/016_storage_encryption.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  change do
+    create_table(:cipher) do
+      column :id, :uuid, primary_key: true, default: Sequel.lit("gen_random_uuid()")
+      column :algorithm, :text, null: false, collate: '"C"'
+      column :key, :bytea, null: false
+      column :init_vector, :bytea, null: true
```

Comment:
You probably want these as encrypted columns, see how raw_private_key_1 is implemented

Reply:
well, I'm not so sure about init vector, I guess that can stay as-is. But the key for sure.

Reply:
I made both encrypted columns